lib/alarmist.ex

```diff
@@ -82,6 +82,16 @@ defmodule Alarmist do
     PropertyTable.subscribe(Alarmist, [alarm_id])
   end
 
+  @doc """
+  Subscribe to alarm status events for all alarms
+
+  See `subscribe/1` for the event format.
+  """
+  @spec subscribe() :: :ok
+  def subscribe() do
```

Comment:
```suggestion
  def subscribe_all() do
```
I keep going back and forth on `subscribe/0` vs. `subscribe_all/0`. My only non-subjective weak argument for adding the `_all` is to compile-time catch someone who forgets to type the argument to `subscribe/1`. Any thoughts?

Reply:
I went back and forth too. Your point about accidentally omitting the argument is enough to push me toward `subscribe_all`.